tests/Unit/Domain/Test_Tags.cpp

```diff
@@ -0,0 +1,36 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "Domain/Tags.hpp"
+
+namespace {
+template <size_t Dim>
+void test() noexcept {
+  CHECK(db::tag_name<Tags::Domain<Dim>>() == "Domain");
```

Comment:
for simple tags please use `TestHelpers::db::test_simple_tag<Tag>("ExpectedNameOfTag");` by including `tests/Unit/DataStructures/DataBox/TestHelpers.hpp`